website/source/docs/guides/acl-migrate-tokens.html.md

```diff
@@ -0,0 +1,415 @@
+the same syntax will break behavior.
+
+Prefix matching can be expressed in the new ACL system explicitly, making the
+following rules in the new system exactly the same as the rules above in the
+old:
+
+```
+node_prefix "foo" { policy = "write" }
+service_prefix "foo" { policy = "write" }
+key_prefix "foo" { policy = "write" }
+```
+
+### API Separation
+
+The "old" API endpoints below continue to work for backwards compatibility but
+will continue to create or show only "legacy" tokens that can't take full
+advantage of the new ACL system improvements. They are documented fully under
+[Legacy Tokens](/api/acl/legacy.html).
+
+- [`PUT /acl/create` - Create Legacy Token](/api/acl/legacy.html#create-acl-token)
+- [`PUT /acl/update` - Update Legacy Token](/api/acl/legacy.html#update-acl-token)
+- [`PUT /acl/destroy/:uuid` - Delete Legacy Token](/api/acl/legacy.html#delete-acl-token)
+- [`GET /acl/info/:uuid` - Read Legacy Token](/api/acl/legacy.html#read-acl-token)
+- [`PUT /acl/clone/:uuid` - Clone Legacy Token](/api/acl/legacy.html#clone-acl-token)
+- [`GET /acl/list` - List Legacy Tokens](/api/acl/legacy.html#list-acls)
+
+The new ACL system includes new API endpoints to manage
+the [ACL System](/api/acl/acl.html), [Tokens](/api/acl/tokens.html)
+and [Policies](/api/acl/policies.html).
+
+## Migration Process
+
+While "legacy" tokens will continue to work for several major releases, it's
+advisable to plan on migrating existing tokens as soon as is convenient.
+Migrating also enables using the new policy management improvements, stricter
+policy syntax rules and other features of the new system without
+re-issuing all the secrets in use.
+
+The high-level process for migrating a legacy token is as follows:
+
+ 1. Create a new policy or policies that grant the required access
+ 2. Update the existing token to use those policies
+
+### Pre-requisites
+
+This process assumes that the 1.4.0 upgrade is complete including all Legacy
+ACLs having their accessor IDs populated. This might take up to several minutes
+after the servers upgrade in the primary datacenter. You can tell if this is the
+case by using `consul acl token list` and checking that no tokens exist with a
+blank `AccessorID`.
+
+In addition, it is assumed that all clients that might _create_ ACL tokens (e.g.
+Vault's Consul secrets engine) have been updated to use the [new ACL
+APIs](/docs/guides/acl-migrate-tokens.html#api-separation).
+
+Specifically if you are using Vault's Consul secrets engine you need to be
+running Vault 1.0.0 or higher, _and_ you must update all roles defined in Vault
+to specify a list of policy names rather than an inline policy (which causes
+Vault to use the legacy API).
+
+~> **Note:** if you have systems still creating "legacy" tokens with the old
+APIs, the migration steps below will still work, however you'll have to keep
+re-running them until nothing is creating legacy tokens to unsure all tokens are
+migrated.
+
+### Creating Policies
```

Comment:
I feel like the voice changes here and becomes rather casual. Not in this round, but later I might want to update it. When I read this, the first thing I think of is; does the order mean anything? Would it be possible to order the creation methods so that the option we feel is the safest/best is at the top? If we could say one is recommended that would be ideal, but I understand there might not actually be one recommended method. Finally, can we say "There are three ways to create new policies" since we cover all three? Overall this section is good too, it is nice to list options with their pros/cons.

Reply:
Yeah my previous step-by-step guides I tried super hard to keep them clinical and in the third person and it got called out (rightfully) as awkward. This may have been over-correcting!

I actually had a few edits I ment to make here so I'll do that and we can see ift here are any obvious ways to improve it maybe?